x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -85,6 +88,8 @@ export const ConfigureCases: React.FC = React.memo(() => {
     isLoading: isPersistingConfiguration,
   } = usePersistConfiguration();
 
+  const isLoadingCustomFields = loadingCaseConfigure || isPersistingConfiguration;
```

Comment:
did this due to a complexity error message thrown by the linter. As it was used twice I could get rid off one complexity level by saving its value here